sci-mathematics/jags/jags-4.0.0.ebuild

```diff
@@ -0,0 +1,47 @@
+# Copyright 1999-2013 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+# $Id$
+
+EAPI=5
+
+inherit autotools-utils toolchain-funcs
+
+MYP="JAGS-${PV}"
+
+DESCRIPTION="Just Another Gibbs Sampler for Bayesian MCMC simulation"
+HOMEPAGE="http://mcmc-jags.sourceforge.net/"
+SRC_URI="mirror://sourceforge/project/mcmc-jags/JAGS/4.x/Source/${MYP}.tar.gz"
+LICENSE="GPL-2"
+IUSE="doc"
+
```

Comment:
I meant move line 16 before line 14 so that you get the following layout

```ebuild
# Copyright 1999-2013 Gentoo Foundation
# Distributed under the terms of the GNU General Public License v2
# $Id$

EAPI

DESCRIPTION
HOMEPAGE
SRC_URI

LICENSE
SLOT
KEYWORDS
IUSE
```

of course this is nothing really important, but when ever you are touching an ebuild try to adopt the general style of the ebuilds in the tree to have more consistency.